it/uk/gov/hmrc/customsservicestatus/helpers/AdditionalAppConfig.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customsservicestatus.helpers
+
+import scala.collection.mutable
+
+trait AdditionalAppConfig {
+  @SuppressWarnings(Array("org.wartremover.warts.MutableDataStructures"))
+  val additionalAppConfig: mutable.Map[String, Any] = mutable.Map.empty
```

Comment:
Can't this live in BaseISpec? I can't see any other implementing classes.

Reply:
it is used https://github.com/hmrc/customs-service-status/blob/RORO-4461/it/uk/gov/hmrc/customsservicestatus/helpers/BaseISpec.scala#L67

Reply:
Yes, that's what I mean - so can't it just be moved to BaseISpec given the "additional app config" in question is just config for integration tests and BaseISpec is extended by all integration tests?

Reply:
At the moment, it's just config for ITs. No need for it to be mutable as things stand either, is there?

Reply:
done.